tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -304,13 +308,72 @@ def _get_effective_padding_node_input(stride, padding,
   return stride * effective_padding_output + padding
 
 
-def compute_receptive_field_from_graph_def(graph_def, input_node, output_node):
+class ReceptiveField:
+  """
+  Receptive field of a convolutioanl neural network.
+
+  Args:
+    size: Receptive field size.
+    stride: Effective stride.
+    padding: Effective padding.
+  """
+  def __init__(self, size, stride, padding):
+    self.size = np.asarray(size)
+    self.stride = np.asarray(stride)
+    self.padding = np.asarray(padding)
+
+  def compute_input_coordinates(self, x, axis=None):
+    """
+    Computes the center of the receptive field that generated a feature.
+
+    Args:
+      x: Coordinate of the feature.
+      axis: The dimensions for which to compute the input coordinates.
```

Comment:
Please explain/document what the default behavior is (ie, what happens if axis is None). My understanding is that it will return a list of centers for each dimension.